ztools/xssh/flag.go

```diff
@@ -90,6 +91,42 @@ func (kaList *KexAlgorithmsList) GetStringSlice() []string {
 	}
 }
 
+type CipherList struct {
+	Ciphers []string
+}
+
+func (cList *CipherList) String() string {
+	return strings.Join(cList.Ciphers, ",")
+}
+
+func (cList *CipherList) Set(value string) error {
```

Comment:
I meant inside the function itself. The spacing looks somewhat arbitrary. Like why is L110 there? I'd personally not put any blank lines given that the function is only 10--20 lines long, but that's up to you as long as there's some rhyme or reason to it.